### PR TITLE
Initial commit for seshat service scripts.

### PR DESCRIPTION
**Pull request by cableramki**

### CODE REVIEW

File: scripts/seshat.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Commenting out for initial Yocto setup
+#[Unit]
+#After=CcspPandMSsp.service CcspMtaAgentSsp.service
```
Comment (schmidtw) [inline scripts/seshat.conf]: Just remove these lines.

File: scripts/seshat.service [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Starts seshatd service
```
Comment (schmidtw) [inline scripts/seshat.service]: Describe what seshatd does -> "seshat service-discovery agent"

File: scripts/seshat.conf (hunk 1 above)
Comment (cableramki) [inline scripts/seshat.conf]: I had left these commented out in order to have an idea of what might need to go in there when getting around to using this file.

File: scripts/seshat.service [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Starts seshat service-discovery agent service
+
+[Service]
+Environment="LOG4C_RCPATH=/rdklogger"
+ExecStart=/usr/bin/seshat -u tcp://192.168.101.3:7777
```
Comment (schmidtw) [inline scripts/seshat.service]: Can the `tcp://192.168.101.3:7777` be `tcp://127.0.0.1:7777` since it's localhost's IP address.

**Comment by cableramki** [comment]
Will re-submit for merging after testing code review feedback.

File: scripts/seshat.service (hunk 3 above)
Comment (cableramki) [inline scripts/seshat.service]: No, because the IP address has to be accessible from ARM also.

**Pull request closed without merging**